.github/workflows/pull-db-tests.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     services:
       pgsql:
-        image: postgres:12
+        image: postgres:16
```

Comment:
Why not 17?

Reply:
Ideally we would run both oldest and newest (13 and 17), but so far there is only one DB version, and 16 is a seemingly good "in between". But if people want 17, we can do 17 of course.